api/src/data_migration/transform.sql

```diff
@@ -0,0 +1,80 @@
+--
+-- Transform queries as SQL.
+--
+
+-- ----------------------------------------------------------------------------
+-- Insert new records.
+WITH transformed_source AS (
+    SELECT
+      f.opportunity_id::BIGINT,
+      f.oppnumber::TEXT AS opportunity_number,
+      f.opptitle::TEXT AS opportunity_title,
+      f.owningagency::TEXT AS agency,
+      CASE f.oppcategory
+        WHEN 'D' THEN 1
+        WHEN 'M' THEN 2
+        WHEN 'C' THEN 3
+        WHEN 'E' THEN 4
+        WHEN 'O' THEN 5
+      END AS opportunity_category_id,
+      f.category_explanation::TEXT AS category_explanation,
+      f.is_draft = 'Y' AS is_draft,
```

Comment:
We actually need a double-negative here.

Is draft in the legacy system isn't quite a boolean, there are actually several values, although even the legacy code merges most of them together and treats it like a boolean. `Y` isn't actually a valid value.

`N` -> False
Anything else -> True

Their java code:

```java
public boolean isDraft() {
		return OppDraftModeType.PUBLISHED != this.oppDraftMode;
	}
```
Where `PUBLISHED` is `N`.